### PR TITLE
Test null keys with containsPair

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by kevmoo** [comment]
Feel free to squash/merge and delete the branch if you're happy

**Comment by kevmoo** [comment]
PTAL!

File: test/map_matchers_test.dart [hunk 1]
```diff
@@ -53,5 +54,34 @@ void main() {
       "Actual: {'a': 1, 'null': null} "
       "Which:  doesn't contain key '2'",
     );
+    shouldFail(
+      {null: null},
+      containsPair('2', 'b'),
+      "Expected: contains pair '2' => 'b' "
+      'Actual: {null: null} '
+      "Which:  doesn't contain key '2'",
+    );
```
Comment (natebosch) [inline test/map_matchers_test.dart:63]: Remove this case, it's covered by lines 79-85 with more descriptive values.

Comment (kevmoo) [reply test/map_matchers_test.dart:63]: ack

**Comment by natebosch** [comment]
You removed the wrong block...

**Comment by kevmoo** [comment]
> You removed the wrong block...

Doh! fixed

**Pull request merged**